Add inode support to quota (#970)
containers/storage: docs/containers-storage.conf.5.md

**ignore_chown_errors** = "false"
  ignore_chown_errors can be set to allow a non privileged user running with a  single UID within a user namespace to run containers. The user can pull and use any image even those with multiple uids.  Note multiple UIDs will be squashed down to the default uid in the container.  These images will have no separation between the users in the container. (default: false)

**inodes**=""

>   Maximum inodes of a container image.   This flag can be used to set quota on the inodes of container images. (format: <number>)

Reviewer: Perhaps?
```suggestion
  Maximum inodes of a container image.   This flag can be used to set a quota on the inodes allocated for a container image. (format: <number>)
```
or "inodes that can be used by a container image."

Reviewer 2: Both are a bit misleading: because we only apply quota limits to read-write layers, images are not affected by quotas.

>   Maximum inodes in a read/write layer.   This flag can be used to set a quota on the inodes allocated for a read/write layer of a container image.

Reviewer 2: ```suggestion
  Maximum inodes in a read/write layer.   This flag can be used to set a quota on the inodes allocated for a read/write layer of a container.
```

>   Maximum inodes in a read/write layer.   This flag can be used to set a quota on the inodes allocated for a read/write layer of a containe rimage. (format: <number>)

Reviewer 3: ```suggestion
  Maximum inodes in a read/write layer.   This flag can be used to set a quota on the inodes allocated for a read/write layer of a container image. (format: <number>[<unit>], where unit = b (bytes), k (kilobytes), m (megabytes), or g (gigabytes))
```

  Comma separated list of default options to be used to mount container images.  Suggested value "nodev". Mount options are documented in the mount(8) man page.

**size**=""

>   Maximum size of a read/write layer.   This flag can be used to set quota on the size of a read/write layer of a container image. (format: <number>[<unit>], where unit = b (bytes), k (kilobytes), m (megabytes), or g (gigabytes))

Reviewer 2: ```suggestion
  Maximum size of a read/write layer.   This flag can be used to set quota on the size of a read/write layer of a container. (format: <number>[<unit>], where unit = b (bytes), k (kilobytes), m (megabytes), or g (gigabytes))
```

>   Maximum size of a read/write layer.   This flag can be used to set quota on the size of a read/write layer of a container images. (format: <number>[<unit>], where unit = b (bytes), k (kilobytes), m (megabytes), or g (gigabytes))

Reviewer 3: ```suggestion
  Maximum size of a read/write layer.   This flag can be used to set quota on the size of a read/write layer of a container image. (format: <number>[<unit>], where unit = b (bytes), k (kilobytes), m (megabytes), or g (gigabytes))
```
